types/ext/core.d.ts

```diff
@@ -22,7 +22,7 @@ export type TypeofResult = 'string' | 'number' | 'bigint' | 'boolean' | 'symbol'
 export type SafeAny = any;
 
 /** This type is used as an explicit way of permitting the `Function` type. */
-// eslint-disable-next-line @typescript-eslint/ban-types
+// eslint-disable-next-line @typescript-eslint/ban-types, @typescript-eslint/no-unsafe-function-type
```

Comment:
This doesn't seem right, ban types is removed in the new typescript-eslint version and this will error